Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -2193,7 +2205,23 @@
             (with-filter (require-not (lockdown-mode))
                 (allow mach-message-send (kernel-mig-routine thread_set_exception_ports)))
 
-            (allow mach-message-send (kernel-mig-routines-common)))))
+            (allow mach-message-send
+                (kernel-mig-routine-only-in-use-during-launch)
```

Comment:
I think it may be more compact to not allow kernel-mig-routine-only-in-use-during-launch here.